environment.yml

```diff
@@ -1,6 +1,7 @@
 name: torchgeo
 channels:
   - conda-forge
+  - pytorch
```

Comment:
@calebrob6 can you test this out and make sure it still installs correctly? If not, we can try adding it to the pip list.

Reply:
Removed torchaudio dependency.